WooCommerce/Classes/POS/Presentation/ItemCardView.swift

```diff
@@ -5,12 +5,28 @@ struct ItemCardView: View {
     private let item: POSItem
     private let onItemCardTapped: (() -> Void)?
 
+    @ScaledMetric private var scale: CGFloat = 1.0
+
     init(item: POSItem, onItemCardTapped: (() -> Void)? = nil) {
         self.item = item
         self.onItemCardTapped = onItemCardTapped
     }
 
     var body: some View {
+        HStack {
+            if let imageSource = item.productImageSource {
+                ProductImageThumbnail(productImageURL: URL(string: imageSource),
+                                      productImageSize: 60,
+                                      scale: scale,
+                                      productImageCornerRadius: 1,
```

Comment:
Should we change the radius to a bigger number or maybe 0? Since 1 is efectively not doing much?

Reply:
Sure! Updated on https://github.com/woocommerce/woocommerce-ios/pull/12966/commits/6fbee5d547be8af199875722726a0d40e018e198 , also moved some of these constants to an extension.